src/tasks/__tests__/processBulkUpload.int.test.ts

```diff
@@ -64,27 +65,39 @@ const createTestBulkUpload = async (
 };
 
 const createTestBaseFields = async (): Promise<void> => {
-	await createBaseField({
-		label: 'Proposal Submitter Email',
-		description: 'The email address of the person who submitted the proposal.',
+	const baseFieldOne = await createBaseField({
```

Comment:
Some day we'll have better test hooks to just... create all this kind of stuff for us in one call (issue already exists: #331)

Reply:
This seems pertinent and like an issue i'd be happy to tackle now that I've been through a refactoring of our testing suite. I'd imagine we'd want some default exports for the values we're populating as well, so we don't have to remember what the test data is supposed to look like